src/lib/components/ProjectCard.svelte

```diff
@@ -1,16 +1,16 @@
 <script lang="ts">
     import type { Project } from '$lib/projects/projects';
-    const { src, project, type, description, href }: Project = $props();
+    const { src, project, description }: Project = $props();
 </script>
 
 <div
-    class="flex h-auto w-full flex-col overflow-hidden rounded-2xl bg-csi-neutral-50 shadow-lg md:h-[32rem] md:w-72 dark:bg-csi-neutral-900"
+    class="flex h-auto w-full flex-col overflow-hidden rounded-2xl bg-csi-neutral-50 px-1 shadow-lg md:max-w-72 md:px-2 dark:bg-csi-neutral-900"
```

Comment:
I've been thinking this through for quite a bit, but I feel like it would be better if card heights along a row are all consistent. At the moment cards with different description lengths or with images look rather awkward next to each other. As an example, see the screenshot of the Projects page below. 

![Screenshot 2024-12-26 at 16-14-24 UP CSI - Center for Student Innovations](https://github.com/user-attachments/assets/8e10df19-dcdb-4703-93ec-f174856f57b9)

Anyhow, I believe this can be quickly resolved by changing `h-auto` to `h-full`, but I'd be keen to hear your thoughts on which looks cleaner.

```suggestion
    class="flex h-full w-full flex-col overflow-hidden rounded-2xl bg-csi-neutral-50 px-1 shadow-lg md:max-w-72 md:px-2 dark:bg-csi-neutral-900"
```

Reply:
Yeah, I do think it looks kind of cleaner. However, I just decided to not use it because there's a huge blank space on cards with no pictures/images (also think of cards with short description and no pictures). I thought it would be look kind of awkward and weird design-wise but, ultimately, I think it still depends on how we format the cards with one another.

Reply:
Yeah, that's fair enough. Personally, between the huge blank spaces on a card or cards with inconsistent sizes, I find that the latter ends up being more awkward in the context of a grid-like panel.